src/component/Header.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import logo from '../assets/logo.png';
+
+const Header = () => {
+  return (
+    <div className="logo_title">
+      <div className="logo">
+        <img alt="logo" src={logo} />
```

Comment:
Please add `title` attribute to all the `img` tags in the project and since it is the project logo, we have the convention to wrap the logo with h1 tag.